src/app/util/ember-compatibility-functions.cpp

```diff
@@ -71,11 +72,15 @@ bool IMEmberAfSendDefaultResponseWithCallback(EmberAfStatus status)
                                                        imCompatibilityEmberAfCluster.commandId,
                                                        (chip::app::CommandPathFlags::kEndpointIdValid) };
 
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
```

Comment:
Are we losing clarity on what time of error this is? Looks like we're dropping resolution of the error, without even logging?

Reply:
currently ember code is not 1on1 matching with IM status code. maybe i can modify the ember code to align with IM ones.